crates/ruff_python_formatter/src/comments/format.rs

```diff
@@ -151,18 +151,43 @@ impl Format<PyFormatContext<'_>> for FormatTrailingComments<'_> {
                 write!(
                     f,
                     [
-                        line_suffix(&format_args![
-                            empty_lines(lines_before_comment),
-                            format_comment(trailing)
-                        ]),
+                        line_suffix(
+                            &format_args![
+                                empty_lines(lines_before_comment),
+                                format_comment(trailing)
+                            ],
+                            0
```

Comment:
It may be worth documenting why using 0 here is fine (because it doesn't matter, so we can avoid doing the computation)

Reply:
Makes sense.

> because it doesn't matter

To make sure I'm on the same page as you, it doesn't matter because there are no cases where an own-line comment's width needs to be measured, correct?

Reply:
Yes, because we don't split comments, and the empty lines expand any enclosing group.

Reply:
Added a comment in 9385e63a278f5e0fbd8f6d78f15605d92e6ba18c but I kinda want to include the detail about splitting as well